Gems/Atom/RHI/Vulkan/Code/Source/Platform/Windows/Vulkan_Traits_Windows.h

```diff
@@ -14,3 +14,4 @@
 #define AZ_TRAIT_ATOM_MOBILE_AZSL_PLATFORM_HEADER "Builders/ShaderHeaders/Platform/Android/Vulkan/PlatformHeader.hlsli"
 #define AZ_TRAIT_ATOM_VULKAN_DISABLE_DUAL_SOURCE_BLENDING 0
 #define AZ_TRAIT_ATOM_VULKAN_RECREATE_SWAPCHAIN_WHEN_SUBOPTIMAL 1
+#define AZ_TRAIT_ATOM_VULKAN_SWAPCHAIN_SCALING_FLAGS AZ::RHI::ScalingFlags::None
```

Comment:
nitpick: file is missing newline at the end.
```suggestion
#define AZ_TRAIT_ATOM_VULKAN_SWAPCHAIN_SCALING_FLAGS AZ::RHI::ScalingFlags::None

```